geopandas/base.py

```diff
@@ -456,6 +456,30 @@ def boundary(self):
         """
         return _delegate_property("boundary", self)
 
+    def build_area(self):
+        """Returns a :class:`~geopandas.GeoSeries` of areal geometries
+        formed by the constituent linework of an input geometry.
+
+        Examples
+        --------
+
+        >>> from shapely.geometry import GeometryCollection, Polygon
+        >>> s = geopandas.GeoSeries([
+        ...     GeometryCollection([
+        ...         Polygon([(0, 0), (3, 0), (3, 3), (0, 3), (0, 0)]),
+        ...         Polygon([(1, 1), (1, 2), (2, 2), (1, 1)])
+        ...     ])
+        ... ])
+        >>> s
+        0    GEOMETRYCOLLECTION (POLYGON ((0.00000 0.00000,...
+        dtype: geometry
+
+        >>> s.build_area()
+        0    POLYGON ((0.00000 0.00000, 0.00000 3.00000, 3....
```

Comment:
What is important on this example from shapely docs is that the second polygon will result in a hole in the first one. That is not clear from this example, since most of our coordinates are not visible.

Reply:
Yes, that's why I wanted to keep this particular example. I could `print` the result of build_area to show the full output?